src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/ReturnValueLocalSymbol.cs

```diff
@@ -29,7 +31,9 @@ internal override BoundExpression RewriteLocal(CSharpCompilation compilation, EE
             var argument = new BoundLiteral(
                 syntax,
                 Microsoft.CodeAnalysis.ConstantValue.Create(_index),
+#nullable disable // can 'method' be null here? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
In general I find i dislike this pattern. Rather than a verbose disable region I think we should be using the null suppression operator. That is designed for exactly this scenario. The comment can just as equally track the `!` here.